sr_utilities/include/sr_utilities/sr_trajectory_command_publisher.hpp

```diff
@@ -0,0 +1,36 @@
+/*
+* Copyright 2011 Shadow Robot Company Ltd.
+*
+* This program is free software: you can redistribute it and/or modify it
+* under the terms of the GNU General Public License as published by the Free
+* Software Foundation version 2 of the License.
+*
+* This program is distributed in the hope that it will be useful, but WITHOUT
+* ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+* FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+* more details.
+*
+* You should have received a copy of the GNU General Public License along
+* with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef _SR_TRAJECTORY_COMMAND_PUBLISHER_HPP_
+#define _SR_TRAJECTORY_COMMAND_PUBLISHER_HPP_
+
+#include <map>
+#include <memory>
```

Comment:
is this used?

Reply:
`shared_ptr` is defined in `<memory>`, see https://en.cppreference.com/w/cpp/memory/shared_ptr